sw/device/tests/rv_core_ibex_nmi_irq_test.c

```diff
@@ -262,9 +257,32 @@ void init_peripherals(void) {
  * - Repeat the previous steps with alert handler.
  */
 bool test_main(void) {
+  // Disable external interrupts via the PLIC entirely.
+  // The NMI should still get through to the processor regardless.
+  irq_global_ctrl(false);
+  irq_external_ctrl(false);
   init_peripherals();
-  alert_handler_nmi_test();
-  wdog_nmi_test();
+
+  // Enable both NMIs.
+  CHECK_DIF_OK(
+      dif_rv_core_ibex_enable_nmi(&rv_core_ibex, kDifRvCoreIbexNmiSourceWdog));
+  CHECK_DIF_OK(
+      dif_rv_core_ibex_enable_nmi(&rv_core_ibex, kDifRvCoreIbexNmiSourceAlert));
+
+  // Execute both NMI tests several times back-to-back to verify that
+  // the causes are successfully cleared in the NMI handlers.
+  for (int k = 0; k < kNumTestIterations; k++) {
```

Comment:
Nit
```suggestion
  for (int k = 0; k < kNumTestIterations; ++k) {
```

Reply:
Is there a specific reason why this is better in this context?
I am just curious, since I have always been using the former style in loops.